requirements.txt

```diff
@@ -1 +1,2 @@
-ops >= 2.2.0
+ops==2.7.0
```

Comment:
May I ask why the specific version pin please? I think there's no need to pin the patch versions :)

Reply:
Reproducible builds :)